com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDSubTarget.cs

```diff
@@ -193,7 +198,10 @@ public override void GetFields(ref TargetFieldContext context)
         {
             // Common properties between all HD master nodes
             // Dots
-            context.AddField(HDFields.DotsInstancing,      systemData.dotsInstancing);
+            context.AddField(HDFields.DotsInstancing, systemData.dotsInstancing);
+
+            // VFX Setup
+            VFXSubTarget.GetFields(ref context);
```

Comment:
VFX also needs to configure fields to enable the primary graph VFX include, as well as setup the necessary VFX output type.